### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by big-andy-coates**
First pass. Only got so far

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,52 +21,37 @@ Instructions are provided for bash scripts (mac)
...
+To run a local Docker based Kafka environment, run through the steps documented in the 
+[CLI quickstart](https://github.com/specmesh/specmesh-build/tree/main/cli#quickstart-using-docker-on-the-local-machine).
 
 Note - the docker environment uses the `kafka_network`
 
-Login to Github Container Registry
-```bash
-% docker login ghcr.io -u <<username>>
->> <<token>>
-```
```
Comment (big-andy-coates) [inline README.md:36]: I've switched the CLI docker image to public, so the login is no longer required.

File: README.md [hunk 2]
```diff
@@ -21,52 +21,37 @@ Instructions are provided for bash scripts (mac)
...
+Checkout a local copy of this quickstart repository, e.g. using either over http
 
+> git clone https://github.com/specmesh/getting-started-apachekafka.git
 
-## 2. Create or modify a SpecMesh spec file `acme_simple_range_life_enhancer-api.yaml` 
+or over ssh
 
-Links: 
-- [resources/acme_simple_range_life_enhancer-api.yaml](resources/acme_simple_range_life_enhancer-api.yaml)
-- Short version [resources/snippet-api.yaml](resources/snippet-api.yaml)
```
Comment (big-andy-coates) [inline README.md:59]: I've removed this short version - it's just confusing. 

Go with one spec - as full / short as you want, but having two is confusing to the user.

File: README.md [hunk 3]
```diff
@@ -21,52 +21,37 @@ Instructions are provided for bash scripts (mac)
...
+> git clone git@github.com:specmesh/getting-started-apachekafka.git
 
-In SpecMesh terms - this file, and what it represents, is considered to be:
-- a data product (Data Mesh terminology)
-- streaming api
-- a policy, or a contract of shared, private and protected related data that is self governed and available to consumers
-- gitops state capture (as part of a git workflow)
-- part of an ecosystem of related dataflow centric apps that are domain centric (each app captures part of the businesses' functionality)
```
Comment (big-andy-coates) [inline README.md:66]: Moved to below the spec

File: README.md [hunk 4]
```diff
@@ -185,25 +170,21 @@ channels:
           $ref: "london.hammersmith.transport._public.tube.passenger.avsc"
 ```
 
-This spec will create 3 topics by prepending the id of the app to the 'owned' topics (channels that use a relative path and start with `_private`, `_protected` or `_public`):
+In SpecMesh terms - this file, and what it represents, is considered to be:
+- a data product (Data Mesh terminology)
```
Comment (big-andy-coates) [inline README.md:174]: I actually disagree here.  I think a spec can define multiple data products.  Each topic can be a data product, not the spec as a whole.

I raised this before when reviewing your doc, and I believe you closed the comment without replying.  Do you disagree?

File: README.md [hunk 5]
```diff
@@ -185,25 +170,21 @@ channels:
...
+- streaming api
+- a policy, or a contract of shared, private and protected related data that is self governed and available to consumers
+- gitops state capture (as part of a git workflow)
+- part of an ecosystem of related dataflow centric apps that are domain centric (each app captures part of the businesses' functionality)
+
+The spec defines 4 topics. Topics are defined under the `channels` section. 
+The spec defines 3 Topics that are _owned_ by this app. 
+Topics that start with `_private`, `_protected` or `_public` are _owned_ topics. The actual topic names will be prefixed with / namespaced under the app's `id`:
```
Comment (big-andy-coates) [inline README.md:182]: I'm a bit confused here.  The app's `id` is actually `urn:acme.simple_range:life_enhancer` - note the use of a colon between the last two parts.  Yet the topic name uses periods in the same place. If this is some magic conversion, rather than a mistake, then it needs explaining here.

File: README.md [hunk 6]
```diff
@@ -185,25 +170,21 @@ channels:
...
-Note, `_protected` also grants (self-governed) access to other principles where a tag is used. For example
-```yaml
-  _protected/purchased:
-     publish:
-        tags: [
-           name: "grant-access:.some.other.principle"
-        ]
-```
+The spec references 1 topic that is _owned_ by another app. Topics owned by another app use a fully qualified name:
+- /london/hammersmith/transport/_public/tube
```
Comment (big-andy-coates) [inline README.md]: This fully qualified name seems at odds with the fully qualified owned topic names above. I would have expected this to at least start with `acme.`. 

Also, why does the spec use '/' vs the actual topic name using `.` separators?  Why not use `/` in both places?

File: README.md [hunk 7]
```diff
@@ -185,25 +170,21 @@ channels:
...
-- granted read-write access to _public prefixed with `acme.simple_range.life_enhancer._public`
-
-Note, `_protected` also grants (self-governed) access to other principles where a tag is used. For example
-```yaml
-  _protected/purchased:
-     publish:
-        tags: [
-           name: "grant-access:.some.other.principle"
-        ]
-```
```
Comment (big-andy-coates) [inline README.md:206]: I've removed this as this demo starts off by saying its NOT going to cover Acls... then talks about them...

File: README.md (hunk 6 above)
Comment (big-andy-coates) [inline README.md]:
```suggestion
- london.hammersmith.transport._public.tube
```

File: README.md [hunk 8]
```diff
@@ -21,52 +21,37 @@ Instructions are provided for bash scripts (mac)
...
 
-# 1. Installation & setup
+Installation & setup
 
-## Running a local Kafka Environment (optional)
+## 1. Running a local Kafka Environment (optional)
 
-See https://github.com/specmesh/specmesh-build/tree/main/cli#quickstart-using-docker-on-the-local-machine
+To run a local Docker based Kafka environment, run through the steps documented in the 
+[CLI quickstart](https://github.com/specmesh/specmesh-build/tree/main/cli#quickstart-using-docker-on-the-local-machine).
```
Comment (big-andy-coates) [inline README.md:29]: Actually, better to inline these

File: README.md (hunk 4 above)
Comment (big-andy-coates) [reply README.md:174]: Make this "one or more data products" ?
